program-list/program-list.json

```diff
@@ -4189,6 +4189,16 @@
     "hall_of_fame": true,
     "safe_harbor": ""
   },
+  {
+    "program_name": "Kubernetes",
+    "policy_url": "https://kubernetes.io/docs/reference/issues-security/security/#report-a-vulnerability",
```

Comment:
We still need to update our docs to point to hackerone. I would just use the H1 policy here
```suggestion
    "policy_url": "https://hackerone.com/kubernetes",
```

Reply:
If you want to keep this pointing to kubernetes.io, I'll update the documentation tomorrow (https://github.com/kubernetes/website/issues/18693)

Reply:
tallclair, I think it makes the most sense given this could change over time, and this would allow you to update this page as that change is occuring.

Reply:
Alright, website docs updated, so we can leave this. Alternatively, there's the shorter alias: https://kubernetes.io/security

Reply:
Updated to kubernetes.io/security